### ISSUE TITLE
Compatible with owncloud 4.5.5

### ISSUE THREAD

**Issue by akiraohgaki** [eskerda/owncloud-submedia-app#13]
I notice error about compatibility but we can continue to use submedia.
Is this just about app version info?

Error log from Admin panel:

```
Error | core | App "SubMedia" (submedia) can't be used because it is not compatible with this version of ownCloud
```

**Comment by eskerda** [comment]
Solved in ba5580efb14ac8ecac8642a3e6cb43d4d475b779

Is this enough to solve the problem? Why is it set to 4.9 precisely?

**Comment by akiraohgaki** [comment]
About 4.0.x series and 4.5.x series

4 (=4.0.x series)
4.9 (=4.5.x series)
4.90.6 (=4.5.5)

We using "oc_share" table, and it is new table since owncloud 4.5, so I just set the same of default apps.
And you can check the owncloud version at owncloud/config/config.php
